frontend/src/js/components/UtilComponents/Token.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { GiantState } from "../../types/redux/GiantState"
+
+import { connect } from "react-redux"
+
+type Props = ReturnType<typeof mapStateToProps>
+
+class Token extends React.Component<Props> {
+    render() {
+        return (
+            <div className="app__main-content">
+                {/* Whistleflow relies on this id */}
+                <div id="giant-api-token">{this.props.auth.jwtToken}</div>
+            </div>
```

Comment:
I was wondering whether it might be worth adding a bit of explainer text in case something goes wrong and the user gets stuck on this page - something like "If you see this page for more than a few seconds, something has gone wrong with securedrop login. Please close this window and try again."

Reply:
Good idea